Code/Framework/AzQtComponents/AzQtComponents/Utilities/Conversions.cpp

```diff
@@ -43,19 +43,20 @@ namespace AzQtComponents
     {
         const QChar decimalPoint = locale.decimalPoint();
         const QChar zeroDigit = locale.zeroDigit();
+        const int numToStringDecimals = numDecimals < 8 ? 8 : numDecimals;
```

Comment:
nit/suggestion: Could this be written as `const int numToStringDecimals = AZStd::max(numDecimals, 8);`?

Reply:
WIll make the change and add a new UnitTest.